pkg/sql/sem/builtins/unaccent_dictionary.go

```diff
@@ -0,0 +1,1519 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package builtins
+
+var unaccentDictionary = map[string]string{
```

Comment:
i'm curious - should this be `map[rune]string`?

Reply:
(if not, I'm not sure how `strings.Split(s, "")` above works with this dictionary.

Reply:
You're right, I have just realized that `strings.Split` may not work correctly. I guess that `map[rune]string` is indeed a way to go. Some "characters" in the dictionary consist of two code points, so I need to modify this dictionary a bit.